packages/gravity/blade-runner/src/analysys/resource-usage.ts

```diff
@@ -25,20 +25,20 @@ export type ResourceUsageStats = {
   }[];
 };
 
-export const analyzeResourceUsage = async (results: PlanResults): Promise<ResourceUsageStats> => {
+export const analyzeResourceUsage = async <S>(results: ReplicantsSummary<S>): Promise<ResourceUsageStats> => {
```

Comment:
i think be careful not to extend the metaphors too much.
results has a clear meaning; i don't know what replicant summary means.
same argument for variable names and types.